### PR TITLE
fix: arch type with downloading dumb-init.

### PR DESCRIPTION
**Pull request by h-r-k-matsumoto**
I'd like to fix a problem #722.

This problem is caused by #710.

### CODE REVIEW

File: runner/Dockerfile [hunk 1]
```diff
@@ -47,7 +47,7 @@ RUN apt update -y \
 # arch command on OS X reports "i386" for Intel CPUs regardless of bitness
 RUN export ARCH=$(echo ${TARGETPLATFORM} | cut -d / -f2) \
     && if [ "$ARCH" = "arm64" ]; then export ARCH=aarch64 ; fi \
-    && if [ "$ARCH" = "i386" ]; then export ARCH=x86_64 ; fi \
+    && if [ "$ARCH" = "amd64" ] || [ "$ARCH" = "i386" ]; then export ARCH=x86_64 ; fi \
```
Comment (ammmze) [inline runner/Dockerfile:50]: We probably need to do this over [here](https://github.com/actions-runner-controller/actions-runner-controller/blob/a9421edd465b327ed6379209087f88119649189e/runner/Dockerfile.dindrunner#L111) too, right?

Comment (h-r-k-matsumoto) [reply runner/Dockerfile:50]: Yes, you're right. 
I'm going to fix it, too :)

Comment (h-r-k-matsumoto) [reply runner/Dockerfile:50]: I've fixed it!
cba8691bebb0b891fa11f1c4b61fdbb101e0b261

File: runner/Dockerfile [hunk 2]
```diff
@@ -47,7 +47,7 @@ RUN apt update -y \
...
     && curl -L -o /usr/local/bin/dumb-init https://github.com/Yelp/dumb-init/releases/download/v${DUMB_INIT_VERSION}/dumb-init_${DUMB_INIT_VERSION}_${ARCH} \
```
Comment (toast-gear) [inline runner/Dockerfile:51]: add the `-f` flag so curl doesn't silently error if it gets a 404

File: runner/Dockerfile.dindrunner [hunk 3]
```diff
@@ -108,7 +108,7 @@ RUN chmod +x /usr/local/bin/startup.sh /usr/local/bin/entrypoint.sh /usr/local/b
 # arch command on OS X reports "i386" for Intel CPUs regardless of bitness
 RUN export ARCH=$(echo ${TARGETPLATFORM} | cut -d / -f2) \
     && if [ "$ARCH" = "arm64" ]; then export ARCH=aarch64 ; fi \
-    && if [ "$ARCH" = "i386" ]; then export ARCH=x86_64 ; fi \
+    && if [ "$ARCH" = "amd64" ] || [ "$ARCH" = "i386" ]; then export ARCH=x86_64 ; fi \
     && curl -L -o /usr/local/bin/dumb-init https://github.com/Yelp/dumb-init/releases/download/v${DUMB_INIT_VERSION}/dumb-init_${DUMB_INIT_VERSION}_${ARCH} \
```
Comment (toast-gear) [inline runner/Dockerfile.dindrunner:112]: add the `-f` flag so curl doesn't silently error if it gets a 404

File: runner/Dockerfile.ubuntu.1804 [hunk 4]
```diff
@@ -47,7 +47,7 @@ RUN apt update -y \
 # arch command on OS X reports "i386" for Intel CPUs regardless of bitness
 RUN export ARCH=$(echo ${TARGETPLATFORM} | cut -d / -f2) \
     && if [ "$ARCH" = "arm64" ]; then export ARCH=aarch64 ; fi \
-    && if [ "$ARCH" = "i386" ]; then export ARCH=x86_64 ; fi \
+    && if [ "$ARCH" = "amd64" ] || [ "$ARCH" = "i386" ]; then export ARCH=x86_64 ; fi \
     && curl -L -o /usr/local/bin/dumb-init https://github.com/Yelp/dumb-init/releases/download/v${DUMB_INIT_VERSION}/dumb-init_${DUMB_INIT_VERSION}_${ARCH} \
```
Comment (toast-gear) [inline runner/Dockerfile.ubuntu.1804:51]: add the `-f` flag so curl doesn't silently error if it gets a 404

**Review by toast-gear — CHANGES_REQUESTED** [review]
In addition, could you add the `-f` to all curl commands in each Dockerfile please

File: runner/Dockerfile.dindrunner (hunk 3 above)
Comment (h-r-k-matsumoto) [reply runner/Dockerfile.dindrunner]: I've added -f option.

File: runner/Dockerfile.ubuntu.1804 (hunk 4 above)
Comment (h-r-k-matsumoto) [reply runner/Dockerfile.ubuntu.1804]: I've added -f option.

**Comment by hammadzz** [comment]
once this PR is merged is the latest image tag now containing the fix?

**Comment by toast-gear** [comment]
@hammadzz yup, should be

**Pull request merged**